pkg/apis/acid.zalan.do/v1/util_test.go

```diff
@@ -298,6 +298,7 @@ var unmarshalCluster = []struct {
 					LoopWait:             10,
 					RetryTimeout:         10,
 					MaximumLagOnFailover: 33554432,
+					SynchronousNodeCount: 1,
```

Comment:
This is not enough here. You have to introduce `synchronous_node_count` option in all the other expected output examples, e.g. search for `maximum_lag_on_failover` and you find the places I mean.

Reply:
Hmm. I introduced `synchronous_node_count` in all places, where i can find `synchronous_mode`.  If i will try to search `maximum_lag_on_failover` there is only one place where i did not intoduced `synchronous_node_count`.
It is *pkg/apis/acid.zalan.do/v1/util_test.go* in a place like [this](https://github.com/Menzorg/postgres-operator/blob/master/pkg/apis/acid.zalan.do/v1/util_test.go#L338) 
But i dont understand is it the right place to introduce `synchronous_node_count`. Could you help, please?